pkg/cmd/repo/gitignore/view/view.go

```diff
@@ -0,0 +1,101 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type ViewOptions struct {
+	IO         *iostreams.IOStreams
+	HTTPClient func() (*http.Client, error)
+	Config     func() (gh.Config, error)
+	Template   string
+}
+
+func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Command {
+	opts := &ViewOptions{
+		IO:         f.IOStreams,
+		HTTPClient: f.HttpClient,
+		Config:     f.Config,
+		Template:   "",
+	}
+
+	cmd := &cobra.Command{
+		Use:   "view <template>",
+		Short: "View an available repository gitignore template",
+		Long: heredoc.Docf(`
+		View an available repository %[1]s.gitignore%[1]s template.
+		
+		%[1]s<template>%[1]s is a case-sensitive %[1]s.gitignore%[1]s template name.
+		`, "`"),
```

Comment:
Aside from indenting the `Long` description, I don't know if positional arguments are formatted this way in other commands 🤔

At the least, we could have a `Examples` section with common gitignore language templates to demonstrate. 🤷

Reply:
>  indenting the Long description

Fixed 👍 

> I don't know if positional arguments are formatted this way in other commands 🤔

Seems they are - can we leave it as-is since there is prescident for this? 

https://github.com/cli/cli/blob/76ea9396275ac5a79d62f6987303b7e12b147ff9/pkg/cmd/codespace/ssh.go#L64

https://github.com/cli/cli/blob/76ea9396275ac5a79d62f6987303b7e12b147ff9/pkg/cmd/run/download/download.go#L53

https://github.com/cli/cli/blob/76ea9396275ac5a79d62f6987303b7e12b147ff9/pkg/cmd/run/rerun/rerun.go#L51

> At the least, we could have a Examples section with common gitignore language templates to demonstrate. 

Yep added examples to `gitignore view` and `license view` as well 👍

Reply:
Nevermind on the second point - I found a different way to word this 👍